grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/MetadataUtil.java

```diff
@@ -52,6 +52,7 @@ public final class MetadataUtil {
     private static final Set<AsciiString> STRIPPED_HEADERS = ImmutableSet.of(
             HttpHeaderNames.STATUS,
             GrpcHeaderNames.GRPC_MESSAGE,
+            GrpcHeaderNames.GRPC_DETAILS,
```

Comment:
Could you also update `MetadataUtilTest`, please?

Reply:
> Could you also update `MetadataUtilTest`, please?

@minwoox thanks for reviewing, I've added a test case